.storybook/preview.tsx

```diff
@@ -15,38 +15,38 @@ import { StorybookHeading } from "./storybookComponents";
 // Custom viewport options
 const customViewports = {
   sm: {
-    name: "sm (320px)",
+    name: "sm - small mobile (less than 480)",
     styles: {
-      width: "320px",
-      height: "568px",
+      width: "390px",
+      height: "667px",
     },
   },
   md: {
-    name: "md (600px)",
+    name: "md - large mobile (less than 768) ",
     styles: {
       width: "600px",
-      height: "800px",
+      height: "715px",
     },
   },
   lg: {
-    name: "lg (960px)",
+    name: "lg - small tablet (less than 1024)",
     styles: {
       width: "960px",
-      height: "800px",
+      height: "1024px",
     },
   },
   xl: {
-    name: "xl (1280px)",
+    name: "xl - large tablet (less than 1280)",
     styles: {
       width: "1280px",
```

Comment:
This may need to be 1279, no?

Reply:
Yes, good point.  Ultimately, there should be no UI differences between the `large mobile` and `desktop` designs and I got hung up on that.  However, after you pointed this out, I see that this should be changed.